Pyrado/pyrado/algorithms/base.py

```diff
@@ -171,6 +174,10 @@ def stopping_criterion_met(self) -> bool:
         """
         return False
 
+    def stopping_criterion_met(self) -> bool:
+        """Checks if the stopping criterion is met."""
+        return self.stopping_criterion(self)
+
```

Comment:
I think there is a better way. How about each algorithm has one self._stopping_criterion, and a getter property for this. The criterion class has a met() or fulfilled() method that returns True or False.